libats-logging/src/main/resources/logback-libats.xml

```diff
@@ -24,7 +24,7 @@
     <logger name="com.zaxxer.hikari" level="${HIKARI_LOG_LEVEL:-INHERITED}" />
     <logger name="slick.backend.DatabaseComponent.action" level="${SLICK_LOG_LEVEL:-INHERITED}" />
     <logger name="com.advancedtelematic" level="${ATS_LOG_LEVEL:-INHERITED}" />
-    <logger name="com.advancedtelematic.libats.messaging.kafka" level="${ATS_KAFKA_LOG_LEVEL:-INHERITED}" />
+    <logger name="com.advancedtelematic.libats.messaging" level="${ATS_MESSAGING_LOG_LEVEL:-INHERITED}" />
```

Comment:
There's more logging going on at this level than in the `kafka` subpackage, so we can handle all of it from here. Alternatively I could create another logger for the subpackage but that doesn't seem worth it.